packages/lingua-franca/scripts/updateAppInsightsGitHubIssue.js

```diff
@@ -1,34 +0,0 @@
-// @ts-check
-
-// Uses the App Insights API to grab useful analytics
-
-// See: https://dev.applicationinsights.io/reference/get-events
-//      https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/57bfeeed-c34a-4ffd-a06b-ccff27ac91b8/resourceGroups/lingua-franca/providers/microsoft.insights/components/TypeScriptLang-Prod-Ai/events
-//
-
-const {
```

Comment:
I actually think we need this stuff if we want to gather statistics. @arengarajan99?